QUARKUS-5667.md

```diff
@@ -0,0 +1,44 @@
+# QUARKUS-5667: Integrate Micrometer with WebSockets Next 
+
+Jira: https://issues.redhat.com/browse/QUARKUS-5667
+
+This feature is specifically Micrometer integration part of the new WebSockets Next extension.
+
+## Scope of the testing
+
+### Upstream testing
+* Unit tests have been added with implementation of this integration. Those
+  unit tests verify that the instrumentation produces correct values for 
+  the following gauges and counters for both client and server:
+  * connection error counter
+  * connection counter
+  * endpoint error counter
+  * inbound/outbound messages counter
+  * inbound/outbound bytes counter
+  * connections closed counter
+
+### General
+* An integration test will be added to Quarkus QE test suite's `monitoring`
+  module. This test will verify that the followingmetrics are correctly 
+  produced for both the websockets next client and server:
+    * connection error counter
+    * connection counter
+    * endpoint error counter
+    * inbound/outbound messages counter
+    * inbound/outbound bytes counter
+    * connections closed counter
+
+### Impact on test suites and testing automation
+* Single test case will be added to `monitoring` module for both bare metal
```

Comment:
Will the test cover both JVM mode and Native mode?

Reply:
Anyway, updated